docs/consuming-rnw.md

```diff
@@ -1,48 +1,55 @@
 ---
 id: consuming-rnw
-title: Consuming React-Native-Windows
+title: Creating a React Native for Windows App
 ---
 
-## Installing React Native
+## Installing React Native Windows
 
-- Install React Native command line interface using NPM:
-  ```
-  npm install -g react-native-cli
-  ```
-- Initialize your React Native project
+There are two different options available when installing a React Native project with the intention of building for Windows: you can either choose the latest **Stable** release or the latest **Beta** release - instructions will call out which steps need to change for either.

```

Comment:
It would be good to clarify that **Stable** implies latest stable vs. **Beta** implies bleeding edge that hasn't been fully vetted yet. Also from below, it appears the Stable is the default - would be good to call that out here.

Reply:
Added more clarity around the beta vs stable releases and their differences.